src/cnc/flavors/aws/shared/provision/partials/collection/providers.tf.j2

```diff
@@ -1,3 +1,4 @@
+{% block shared_functions %}
```

Comment:
I think we can rm this shared_functions block ^

Reply:
Commit: https://github.com/coherenceplatform/cnc/pull/90/commits/bcc63fab49021df257b003f403c76105bf4a6d6d